README.md

```diff
@@ -1,2 +1,38 @@
-# map-app
-A map app for beginners in the OSMP 2015 program.
+
+##Aging Out of Place?
+**OSMP 2015 - A visual analysis showing the impact of gentrification on Philadelphia’s elderly population.**
+
+As a “city of neighborhoods”, much of Philadelphia’s culture has been established by long term residents. As the city has finally begun to see a population increase, attention has been focused on the influx of millennials and young families living in (or returning to) Philadelphia. However, less analysis has been completed to determine how these current trends have impacted (or will soon impact) the city’s elderly population. 
+
+Currently, Philadelphia ranks 12th in the nation among areas experiencing a boom in gentrification. Growth in Philadelphia has not been widespread and is limited mostly to only a few select neighborhoods, such as West, South and Lower North Philadelphia. Anecdotally, many have claimed that new development is displacing low-income and elderly residents, so this project seeks to examine what geographic patterns exists. 
+
+The project will build upon the existing Philadelphia gentrification maps and look at the change in the elderly populations (if any) within the tracts that were identified as “gentrifying”. This project will also include other statistics about aging and end-of-life issues that could be impacted by changes in urban development, such as transportation costs, access to health care, etc.
+
+###Definition of Gentrification
+There is no agreed upon definition for gentrification. This project will utilize the existing definition from the Governing Data report (see below in sources) exploring gentrification in major metropolitan areas. As defined by that study, gentrification was measured by the following Census and American Community Survey categories:
+-increased median household income
+-significant growth in median home value
+-improvements in educational attainment
+
+*For more information on their methodology, please refer to:*
+http://www.governing.com/gov-data/gentrification-report-methodology.html
+
+###Data Sources:
+[Pew Research Center Survey on Aging and Longevity (2013)] (http://www.pewforum.org/datasets/survey-of-aging-and-longevity/)
+
+[Governing - Gentrification in America Report] (http://www.governing.com/gov-data/gentrification-in-cities-governing-report.html)
+
+[Governing - Philadelphia Gentrification Maps] (http://www.governing.com/gov-data/philadelphia-gentrification-maps-demographic-data.html)
+
+[2000 US Census](http://www.census.gov/main/www/cen2000.html)
+
+[2009-13 American Communities Survey] (https://www.census.gov/programs-surveys/acs/data.html)
+
+[Age Demographics - Statistics for US Counties] (www.governing.com/gov-data/age-65-older-population-map-usa-counties.html)
+
+[PCAC Development without Displacement Report](http://phillyaffordablecommunities.org/wp-content/uploads/2015/03/DWD_Report.pdf)
+
+
```

Comment:
Just a heads up that a bunch of returns like this don't get respected by Markdown. If you really wanted to inject spacing, you'd use `<br />` (also known as a break tag). That said, use of break tags in Markdown is rare. In this scenario, I think things look fine without extra spacing.

See also: http://daringfireball.net/projects/markdown/syntax#block